networking/ovn_kubernetes_network_provider/configuring-egress-ips-ovn.adoc

```diff
@@ -12,18 +13,31 @@ include::modules/nw-egress-ips-about.adoc[leveloffset=+1]
 
 include::modules/nw-egress-ips-object.adoc[leveloffset=+1]
 
+ifndef::openshift-rosa[]
 include::modules/nw-egress-ips-config-object.adoc[leveloffset=+1]
+endif::openshift-rosa[]
 
 include::modules/nw-egress-ips-node.adoc[leveloffset=+1]
 
 [id="configuring-egress-ips-next-steps"]
 == Next steps
 
+ifndef::openshift-dedicated,openshift-rosa[]
 * xref:../../networking/ovn_kubernetes_network_provider/assigning-egress-ips-ovn.adoc#assigning-egress-ips-ovn[Assigning egress IPs]
+endif::openshift-dedicated,openshift-rosa[]
+ifdef::openshift-dedicated,openshift-rosa[]
+* link:https://docs.openshift.com/container-platform/4.14/networking/ovn_kubernetes_network_provider/assigning-egress-ips-ovn.html#assigning-egress-ips-ovn[Assigning egress IPs]
+endif::openshift-dedicated,openshift-rosa[]
 
 [role="_additional-resources"]
 [id="configuring-egress-ips-additional-resources"]
 == Additional resources
 
+ifndef::openshift-dedicated,openshift-rosa[]
 * xref:../../rest_api/objects/index.adoc#labelselector-meta-v1[LabelSelector meta/v1]
 * xref:../../rest_api/objects/index.adoc#labelselectorrequirement-meta-v1[LabelSelectorRequirement meta/v1]
+endif::openshift-dedicated,openshift-rosa[]
+ifdef::openshift-dedicated,openshift-rosa[]
+* link:https://docs.openshift.com/container-platform/4.14/rest_api/objects/index.html#labelselector-meta-v1[LabelSelector meta/v1]
+* link:https://docs.openshift.com/container-platform/4.14/rest_api/objects/index.html#labelselectorrequirement-meta-v1[LabelSelectorRequirement meta/v1]
```

Comment:
Just want to make sure that the intent is to actually link into the 4.14 docs at lines 29, 41, and 42. I am not familiar with the ROSA doc set, but wanted to ask as this could end up being a maintenance item for you.

Reply:
Yup, this works. We have to link to OCP directly, and we use GA+1.